mtgjson5/set_builder.py

```diff
@@ -683,6 +685,10 @@ def build_mtgjson_card(
         if card_type not in {"starterdeck", "planeswalkerdeck"}
     ]
 
+    card_release_date = scryfall_object.get("released_at")
+    if set_release_date != "" and set_release_date != card_release_date:
```

Comment:
This can be shortened to 
```suggestion
    if set_release_date and set_release_date != card_release_date:
```